src/models/user.js

```diff
@@ -37,6 +37,13 @@ export default (sequelize, DataTypes) => {
   }
 
   // Class methods
+  User.getNonSensibleInformation = async function (user) {
+    return {
+      username: this.username,
```

Comment:
In this context, `this` represents the `User` class (not an instance). So, you should replace `this` by `user`.

Reply:
I wrote it that way the first time and it would not work. I even tried to use the user as an argument but still nothing.